LibertyDSNP/example-client
# setup basic component structure and formatting

# Problem

As a developer, I want to have the component structure roughly setup, along with basic formatting, so that when we build out the first features, there is a sense of organization.
[link to Pivotal Tracker #12345678](https://www.pivotaltracker.com/story/show/12345678)

# Solution
- Build the general component structure
- Add formatting only
 - No brand-influenced styling
- Add tests to make sure components render

## Change summary:

- Created the following empty components
 - Header
 - Feed
 - NewPost
 - PostList
 - Profile
- Created tests to make sure they render
- Added styling to for layout and to show layout

## Steps to Verify:

1. Run EC and see that everything is rendering correctly
2. Check that the component structure is correct

## Screenshots (optional):
<img width="1539" alt="Screen Shot 2021-03-19 at 4 17 10 PM" src="https://user-images.githubusercontent.com/43625033/111850861-aa7ea880-88ce-11eb-8165-7c3a191729f4.png">
